limitador-server/src/main.rs

```diff
@@ -210,6 +210,9 @@ impl Limiter {
                             Self::Blocking(limiter) => limiter.configure_with(limits)?,
                             Self::Async(limiter) => limiter.configure_with(limits).await?,
                         }
+                        if limitador::limit::check_deprecated_syntax_usages_and_reset() {
```

Comment:
if the feature is not enabled, this would not compile

Reply:
Sure... but the feature is always enabled. The way I've done this is the crate has the feature, and this enables it _always_ as a consumer of the crate.
Are you asking I do make the feature available at the binary's layer too (like I did for infinispan)?

Reply:
ok I see.

The feature belongs to the `limitador` crate. And the limitador crate compiles regardless of being activated or not. However, limitador-server package needs it activated or it would not compile.

LGTM